### DISCUSSION TITLE
gpd.read_file(gpd.datasets.get_path('naturalearth_lowres')) throws error: Symbol not found: (_ZSTD_compressBound)

### DISCUSSION THREAD

**Discussion by jolespin** [geopandas/geopandas#2885] [Installation]
- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the latest version of geopandas.

- [ ] (optional) I have confirmed this bug exists on the main branch of geopandas.

---

**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

#### Code Sample, a copy-pastable example

```python
world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))

ImportError                               Traceback (most recent call last)
Input In [13], in <module>
      1 df_coords = pd.DataFrame(df_meta_samples["Latitude and Longitude"].apply(lambda x: list(format_coords(x))).to_dict(), index=["Latitude", "Longitude"]).T
      3 gdf_coords = gpd.GeoDataFrame(data=df_coords, geometry=gpd.points_from_xy(x=df_coords["Longitude"], y=df_coords["Latitude"]), index=df_coords.index).set_crs("EPSG:4326")
----> 5 world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
      6 mmin = np.min(gdf_coords[["Latitude", "Longitude"]].values.ravel())
      7 mmax = np.max(gdf_coords[["Latitude", "Longitude"]].values.ravel())

File ~/anaconda3/envs/soothsayer_py3.9_env/lib/python3.9/site-packages/geopandas/io/file.py:242, in _read_file(filename, bbox, mask, rows, engine, **kwargs)
    172 def _read_file(filename, bbox=None, mask=None, rows=None, engine=None, **kwargs):
    173     """
    174     Returns a GeoDataFrame from a file or URL.
    175 
   (...)
    240     by using the encoding keyword parameter, e.g. ``encoding='utf-8'``.
    241     """
--> 242     engine = _check_engine(engine, "'read_file' function")
    244     filename = _expand_user(filename)
    246     from_bytes = False

File ~/anaconda3/envs/soothsayer_py3.9_env/lib/python3.9/site-packages/geopandas/io/file.py:112, in _check_engine(engine, func)
    110     _check_pyogrio(func)
    111 elif engine is None:
--> 112     raise ImportError(
    113         f"The {func} requires the 'pyogrio' or 'fiona' package, "
    114         "but neither is installed or imports correctly."
    115         f"\nImporting fiona resulted in: {fiona_import_error}"
    116         f"\nImporting pyogrio resulted in: {pyogrio_import_error}"
    117     )
    119 return engine

ImportError: The 'read_file' function requires the 'pyogrio' or 'fiona' package, but neither is installed or imports correctly.
Importing fiona resulted in: dlopen(/Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/lib/python3.9/site-packages/fiona/ogrext.cpython-39-darwin.so, 0x0002): Symbol not found: (_ZSTD_compressBound)
  Referenced from: '/Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/lib/libgdal.30.dylib'
  Expected in: '/Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/lib/libblosc.1.21.2.dylib'
Importing pyogrio resulted in: No module named 'pyogrio'

```

#### Problem description

Failed loading backend programs.  I've uninstalled and reinstalled both pyogrio and fiona. 

#### Output of ``geopandas.show_versions()``

<details>

SYSTEM INFO
-----------
python     : 3.9.15 | packaged by conda-forge | (main, Nov 22 2022, 08:55:37)  [Clang 14.0.6 ]
executable : /Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/bin/python
machine    : macOS-12.6-x86_64-i386-64bit

GEOS, GDAL, PROJ INFO
---------------------
GEOS       : 3.10.2
GEOS lib   : /Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/lib/libgeos_c.dylib
GDAL       : 3.5.0
GDAL data dir: None
PROJ       : 9.0.0
PROJ data dir: /Users/jespinoz/anaconda3/envs/soothsayer_py3.9_env/share/proj

PYTHON DEPENDENCIES
-------------------
geopandas  : 0.12.2
numpy      : 1.21.5
pandas     : 1.4.0
pyproj     : 3.3.1
shapely    : 1.8.2
fiona      : None
geoalchemy2: None
geopy      : 2.3.0
matplotlib : 3.5.1
mapclassify: 2.5.0
pygeos     : 0.12.0
pyogrio    : v0.4.1
psycopg2   : None
pyarrow    : None
rtree      : 1.0.1
</details>

**Comment by martinfleis** [comment]
How did you install Fiona and pyogrio? The Fiona issue shown above is a corrupted installation of GDAL. You haven't shown a pyogrio issue, so hard to tell there.

**Reply by jolespin** [reply]
Is there anything else I can provide? I install geopandas and geopy through conda so I believe fiona and pyogrio were in the recipe and installed via conda.

**Reply by martinfleis** [reply]
pyogrio is not in the recipe by default, you need to install it separately. I would try to reinstall `gdal` and `fiona`. I don't think that the issue is caused by terminating the execution as you suggest.

**Comment by jolespin** [comment]
I think I know what caused this.  I imported this for the first time in a Jupyter cell and realized there was more code than I meant to run.  I canceled it and then it caused the error.  I made a separate conda environment just for geopandas and geopy and everything installed correctly enough to run the import. 

Do you know what I can reinstall?
